www/fedp04/cv01/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+$jmeno = 'Polina';
+$prijmeni = 'Fediaeva';
+$vek = '04.08.2002';
+$adresa = 'Praha 4';
+$telefon = '1234567';
+$pozice = 'Director of the firm';
+$nazev_firmy = 'Best firm';
+$email = 'zzefir.rum@mail.com';
+$web = 'www.web.com';
+$prace = 'Available for job';
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Business_card</title>
+    <link href='https://fonts.googleapis.com/css?family=Merienda' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+
+<body>
+    <div class="business-card-container">
+
+        <div class="business-card front">
+
+            <div class="front-image"><img src=<?php echo "./pepeRound.png" ?>></div>

```

Comment:
Je lepsi ten `pepeRound.png` dat to promenne a pak ji volat pomoci echo, jinak ten `echo "./pepeRound.png"` nedava smysl podle me.